libcxx/include/variant

```diff
@@ -995,7 +995,7 @@ class _LIBCPP_TEMPLATE_VIS __copy_assignment;
     template <class... _Types>                                                                                         \
     class _LIBCPP_TEMPLATE_VIS __copy_assignment<__traits<_Types...>, copy_assignable_trait>                           \
         : public __move_assignment<__traits<_Types...>> {                                                              \
-      using __base_type = __move_assignment<__traits<_Types...>>;                                                      \
+      using __base_type [[__gnu__::__nodebug__]] = __move_assignment<__traits<_Types...>>;                             \
```

Comment:
I'm generally fine with this change, but I'd like us to stick with `_LIBCPP_NODEBUG` since we have precedent for it. If we want to discuss dropping `_LIBCPP_NODEBUG` in favor of using raw attributes, that should be pursued separately.